contracts/Protected.sol

```diff
@@ -69,19 +69,25 @@ contract Protected {
         bytes32 _id,
         address _to,
         bool _assignable,
-        uint _expiration,
-        uint _uses
+        uint80 _startTime,
+        uint80 _expiration,
+        uint80 _uses
     ) public
     {
         Key memory key = keys[_id][msg.sender];
         require(key.exists && isValidExpiration(key.expiration), "Invalid key");
         require(key.assignable, "Key is not assignable");
+        require(key.startTime == 0 || (_startTime >= key.startTime && _startTime > 0), "Cannot reduce key's start time");
         require(key.expiration == 0 || (_expiration <= key.expiration && _expiration > 0), "Cannot extend key's expiration");
-        require(key.uses == 0 || (_uses <= key.uses && _uses > 0), "Not enough uses avaiable");
+        require(_startTime == 0 || _expiration == 0 || _startTime < _expiration, "Start time must be strictly less than expiration");
         require(isValidExpiration(_expiration), "Expiration must be in the future");
+        require(_startTime == 0 || _startTime >= now, "Start time must be in the future");
+        require(key.uses == 0 || (_uses <= key.uses && _uses > 0), "Not enough uses avaiable");
 
+        bool possesKey = (!unlockable(_id, _to) && keys[_id][_to].startTime <= now);
         require(
-            !unlockable(_id, _to) || (keys[_id][_to].assignable == _assignable && keys[_id][_to].expiration == _expiration),
+            possesKey ||
+            (keys[_id][_to].assignable == _assignable && keys[_id][_to].expiration == _expiration && keys[_id][_to].startTime == _startTime),
```

Comment:
I think that a key can still be merged safely if both start times are in the past.